docs/netlify_deploy.md

```diff
@@ -0,0 +1,38 @@
+# Netlify deploy process
+
+We use Netlify to deploy clayui.com and a version of Storybook, the deployment process is very simple, the build happens in the service itself so we don't have a Github Actions configuration.
+
+A new version of clayui.com and storybook are triggered in different scenarios.
+
+-   **Deploy Previews** - every new PR a new build is made to show the changes made
+-   **next.clayui.com** - Represents changes that are in the master branch
+-   **next.storybook.clayui.com** - Represents changes that are in the master branch
+-   **clayui.com** - Represents changes that are in the stable branch
+-   **storybook.clayui.com** - Represents changes that are in the stable branch
+
+## Deploy Previews
+
+All PR triggers a new build for clayui.com and storybook and generates a preview URL that allows you to preview the changes without having to download the PR locally and test.
+
+Netlify uses Github Checks to demonstrate the build status, when the build is being done it is possible to click `Details` on the corresponding check and check the build status as well as the logs. When done clicking on `Details` again will redirect to the corresponding preview that was generated.
+
+> No need to be a member to view build status or preview
+
+<img width="916" alt="Screen Shot 2022-02-08 at 2 09 11 PM" src="https://user-images.githubusercontent.com/13750819/153067419-b0dad542-9d4b-4bdc-9378-aa7100d5a230.png">
+
+[Learn more about Deploy Previews to Netlify](https://docs.netlify.com/site-deploys/deploy-previews/)
+
+## Configuration
+
+Netlify configuration for clayui.com and storybook is very simple, we don't need extra configuration for these cases or Github Actions for example, much of the configuration is done the first time we integrate with Netlify and connect the account with the Github repository.
+
+Netlify will observe all changes in the repository and trigger builds when necessary, in the above cases for example. It has its own build system, so we need to tell which command to run and it will do everything else.
```

Comment:
```suggestion
Netlify will observe all changes in the repository and trigger builds when necessary. We just need to specify which command to run and it will do  the rest.
```